### PR TITLE
feat: add mobile emulation to /learn section

### PR DESCRIPTION
**Pull request by stefanjudis**
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [ ] Docs
* [x] Learn
* [ ] Other

Add a new section to `/learn` to cover mobile emulation.

### CODE REVIEW

File: site/content/learn/headless/basics-emulating-mobile-devices.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  - basics
+
+weight: 8
+
+menu:
+  learn:
+    parent: "Getting started"
+---
+
+Puppeteer and Playwright control headless desktop browsers but can also emulate mobile devices. And while mobile device emulation can't replace testing on mobile devices, it's a practical and quick-to-setup approach to test your site in a mobile scenario.
```
Comment (ragog) [inline site/content/learn/headless/basics-emulating-mobile-devices.md:21]: @stefanjudis do you think we could spell out why it still makes sense to use this approach, given how it doesn't replace testing on mobile? Can we claim it is still enough to test responsiveness? The question that pops in my mind after reading this intro is "ok it's easy to set up but is not the real thing... how close to real is it though? what _can_ I do with it?"

**Review by ragog — DISMISSED** [review]
@stefanjudis I left a nitpick to see if we can improve the intro but if not np, LGTM

File: site/content/learn/headless/basics-emulating-mobile-devices.md (hunk 1 above)
Comment (stefanjudis) [reply site/content/learn/headless/basics-emulating-mobile-devices.md:21]: Great feedback. let's improve!

**Review by ragog — APPROVED** [review]
LGTM

**Pull request merged**